public/survey-create.html

```
@@ -0,0 +1,479 @@
+          <h3 id="confirm-modal-title" class="font-bold text-gray-800 dark:text-gray-200">Clear draft?</h3>
+          <p id="confirm-modal-message" class="text-sm text-gray-500 dark:text-gray-400 mt-1">This cannot be undone.</p>
+        </div>
+      </div>
+      <div class="flex justify-end gap-2">
+        <button type="button" onclick="closeConfirmModal()" class="px-4 py-2 rounded-lg text-sm font-semibold text-gray-600 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-gray-700 transition">Cancel</button>
+        <button type="button" id="confirm-modal-confirm-btn" class="px-4 py-2 rounded-lg text-sm font-semibold bg-red-600 hover:bg-red-700 text-white transition">Clear Draft</button>
+      </div>
+    </div>
+  </div>
+
+  <div id="site-footer"></div>
```

HTMLHint (1.9.2)

[error] The id value [ site-footer ] must be unique.

(id-unique)